docs/_data/contributors/contributors-jupyterhub.yaml

```
@@ -1,113 +1,128 @@
-# Use ALPHABETICAL ORDER please :-)
-- name: Matthias Busonnier
-  handle: "carreau"
-  affiliation: UC Merced
-  contributions: code,infra
-  status: active
-  last-check-in: 2019-10
+# teams:
+# - council: JupyterHub council
+# - maintainers:
+# - mybinder: mybinder.org operators
+# no team means inactive
 
+# Use ALPHABETICAL ORDER please :-)
 - name: Georgiana Dolocan
   handle: "GeorgianaElena"
-  affiliation: Simula
+  affiliation: 2i2c
   contributions: code
   focus: jupyterhub
-  status: active
-  last-check-in: 2019-10
-
-- name: Jessica Forde
-  handle: "jzf2101"
-  affiliation: UC Berkeley
-  contributions: doc
-  focus: jupyterhub, binder
-  status: active
-  last-check-in: 2019-10
+  teams:
+    - council
+  last-check-in: 2022-09
 
 - name: Sarah Gibson
   handle: "sgibson91"
   affiliation: 2i2c
   contributions: question,doc,tutorial,infra,talk,test
   focus: jupyterhub, binder
-  status: active
-  last-check-in: 2019-10
+  teams:
+    - council
+  last-check-in: 2022-09
 
 - name: Tim Head
   handle: "betatim"
   affiliation: Wild Tree Tech
   contributions: bizdev, code, fundingFinding, ideas, talk, infra
   focus: jupyterhub, binder
-  status: active
-  last-check-in: 2019-12
+  teams:
+    - council
+  last-check-in: 2022-09
 
 - name: Chris Holdgraf
   handle: "choldgraf"
   affiliation: 2i2c
   contributions: code,ideas,doc,question
   focus: jupyterhub, binder
-  status: active
-  last-check-in: 2019-10
+  teams:
+    - council
+  last-check-in: 2022-09
 
 - name: Simon Li
   handle: "manics"
   affiliation: Open Microscopy Environment, University of Dundee
```

```suggestion
  affiliation: University of Dundee
```
I moved departments recently, easier to keep this generic.